focus_converter_base/focus_converter/conversion_configs/gcp/1_commitment_discount_name_S001.yaml

```diff
@@ -0,0 +1,7 @@
+plan_name: Unnest credits.full_name to get CommitmentDiscountName
+conversion_type: unnest
+column: credits.id
```

Comment:
Should the **column** name be credits.full_name instead of credits.id ?

Reply:
Thankyou @spriharani, also we have to deal with the array values, do you have a recommendation how we do that?
Since credits object has multiple values.